release/internal/pinnedversion/pinnedversion.go

```diff
@@ -286,19 +301,37 @@ func LoadHashrelease(repoRootDir, outputDir, hashreleaseSrcBaseDir string, lates
 	}, nil
 }
 
+// RetrieveImageComponents retrieves the images from Calico components in the pinned version file that produce images.
+// It also adds the Tigera operator and its init image to the returned map.
+//
+// Images are expected to be in the format "<registry>/<image-name>" where <registry> includes the registry and image path.
```

Comment:
> Images are expected to be in the format "<registry>/<image-name>"

Does this mean images in the versions file are expected to be of this format? Or the images returned from this function are expected to be in this format? 

> registry> includes the registry and image path.

This is self-referential. Can we define registry without using the word registry?